android/src/main/java/com/shiftsmart/RNZenDeskSupport/HelpCenterActivityBuilder.java

```diff
@@ -55,25 +55,25 @@ SupportActivityBuilder withOptions(ReadableMap options) {
       if (options.hasKey("withContactUsButtonVisibility")) {
         switch(options.getString("withContactUsButtonVisibility")) {
           case "OFF":
-            withContactUsButtonVisibility(ContactUsButtonVisibility.OFF);
```

Comment:
Are we sure we need to swap all these to manual booleans? Seems like they could be some `configuration` values set via the ZenDesk UI 🤔

Reply:
for this version of the SDK it's defined on a ActivityBuilder level